storybook/src/docs/space.docs.mdx

```diff
@@ -6,7 +6,26 @@ import { Meta } from "@storybook/blocks";
 
 # Space
 
-We distinguish 4 types of white space: Grid, Stack, Inside, and Type.
+We distinguish 3 types of white space: Component, Grid, and Type.
+
+## Component Space
+
+This is the space between components, e.g. between a list of cards, a row of buttons, or an icon and an accompanying text.
+
+Or at the four edges of a component: at the top, bottom, start, and end of its containing box,
+as used for buttons and input fields, and for larger components with a coloured background or border.
+
+This type of space is based on the font size – the medium space length equals the font size.
```

Comment:
This reads a little weird. There are multiple font sizes, right? We could do something like:

```suggestion
This type of space is based on the font size – the medium space length equals the default Paragraph font size.
```